README.md

```diff
@@ -24,23 +24,28 @@ $ brew install awscli
 ```
 $ aws --endpoint-url=http://localhost:4566 s3 mb s3://reddit-soccer-goals
 ```
-6. Run db migrations
+6. Install golang-migrate
+```
+brew install golang-migrate
```

Comment:
Our other commands in the readme have $ in front of them. Idk why but we should be consistent and add it here or remove from the others